src/codegen.cpp

```diff
@@ -8783,7 +8783,9 @@ static jl_llvm_functions_t
             }
         }
     }
-
+    jl_aliasinfo_t ai = jl_aliasinfo_t::fromTBAA(ctx, ctx.tbaa().tbaa_gcframe);
+    ctx.world_age_value = ai.decorateInst(
```

Comment:
I think you can just use `closure_world.V` from the OpaqueClosure store above - no need actually perform the load

Reply:
That load there only exists when doing opaque closures, and we want this code to work always, also I just realized I need to move this load a bit lower

Reply:
Right, I think loading early and then doing the swap in the OpaqueClosure close would work for all code - that way you can use it for OC load of the old world too